firebase-appdistribution/src/main/res/values/strings.xml

```diff
@@ -25,6 +25,10 @@
   <string name="downloading_app_update">Downloading in-app update...</string>
   <string name="download_completed">Download completed</string>
   <string name="download_failed">Download failed</string>
+  <string name="install_failed">Install Failed</string>
```

Comment:
nit: Could we make this casing consistent with the other error messages? (The first letter capitalized, the rest lower case)

Reply:
good catch! thanks!